test/test_filesystem_helper.cpp

```diff
@@ -14,16 +14,42 @@
 
 #include <gtest/gtest.h>
 
+#include <fstream>
 #include <string>
 
 #include "rcpputils/filesystem_helper.hpp"
+#include "temporary_directory_fixture.hpp"
 
 #ifdef _WIN32
 static constexpr const bool is_win32 = true;
```

Comment:
```c++
namespace
{
#ifdef _WIN32
constexpr const bool is_win32 = true;
#else
constexpr const bool is_win32 = false;
#endif
}
```

Reply:
This is something that was already there and not sure if it's something we want to touch in this PR.

Reply:
We shouldn't touch it if it was there before.